sdk/cosmos/tests.live.yml

```diff
@@ -0,0 +1,429 @@
+trigger:
```

Comment:
You want to name this one tests.yml and your other one ci.yml to match our conventions.

Reply:
done